src/NUnitFramework/framework/Internal/AsyncToSyncAdapter.cs

```diff
@@ -22,13 +22,19 @@ public static bool IsAsyncOperation(Delegate @delegate)
         }
 
         public static object? Await(Func<object?> invoke)
-            => Await<object?>(invoke);
+            => Await<object?>(default(TestExecutionContext), invoke);
 
         public static TResult? Await<TResult>(Func<object?> invoke)
+            => Await<TResult>(default(TestExecutionContext), invoke);
```

Comment:
(optional) So as not to imply that TestExecutionContext is a struct:
```suggestion
            => Await<TResult>((TestExecutionContext)null, invoke);
```

Reply:
Removed overload, so no longer relevant.

I don't associate `default(T)` with struct, just a different (nicer?) way to write `(T)null`